pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
Agreed we should delete this line. If a CI job publishes FindBugs results and sets build status to `UNSTABLE` then it may disable failure on the Maven command line.